src/styles/main.scss

```diff
@@ -1,3 +1,218 @@
-body {
+$blue: #00acdc;
+$dark-blue: #060b35;
+$gray: #616070;
+$white: #fff;
+$font-family: "Roboto", sans-serif;
+
+* {
   margin: 0;
+  font-family: $font-family;
+}
```

Comment:
don't reset margins for `*`. Use tags directly.
Set font-family for html or body selector